src/command_modules/azure-cli-eventhubs/azure/cli/command_modules/eventhubs/_params.py

```diff
@@ -0,0 +1,119 @@
+
+    with self.argument_context('eventhubs namespace') as c:
+        c.argument('namespace_name', arg_type=name_type, id_part='name', help='name of the Namespace')
+
+    with self.argument_context('eventhubs namespace create') as c:
+        c.argument('tags', arg_type=tags_type)
+        c.argument('sku', options_list=['--sku-name'], arg_type=get_enum_type(SkuName))
+        c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
+        c.argument('skutier', options_list=['--sku-tier'], arg_type=get_enum_type(SkuTier))
+        c.argument('capacity', type=int, help='Capacity for Sku')
+        c.argument('is_auto_inflate_enabled', arg_type=get_three_state_flag(), help='Value that indicates whether AutoInflate is enabled for eventhub namespace.')
+        c.argument('maximum_throughput_units', type=int, help='Upper limit of throughput units when AutoInflate is enabled, vaule should be within 0 to 20 throughput units. ( 0 if AutoInflateEnabled = true)')
+
+    with self.argument_context('eventhubs namespace update') as c:
+        c.argument('namespace_name', arg_type=name_type, id_part='name', help='name of the Namespace')
+        c.argument('tags', arg_type=tags_type)
+        c.argument('sku', options_list=['--sku-name'], arg_type=get_enum_type(SkuName))
+        c.argument('skutier', options_list=['--sku-tier'], arg_type=get_enum_type(SkuTier))
+        c.argument('capacity', type=int, help='Capacity for Sku')
+        c.argument('is_auto_inflate_enabled', arg_type=get_three_state_flag(), help='Value that indicates whether AutoInflate is enabled for eventhub namespace.')
+        c.argument('maximum_throughput_units', type=int, help='Upper limit of throughput units when AutoInflate is enabled, vaule should be within 0 to 20 throughput units. ( 0 if AutoInflateEnabled = true)')
+
+    # region Namespace Authorizationrule
+    for scope in ['eventhubs namespace authorization-rule', 'eventhubs namespace authorization-rule keys list', 'eventhubs namespace authorization-rule keys renew']:
+        with self.argument_context(scope) as c:
+            c.argument('authorization_rule_name', arg_type=name_type, id_part='child_name_1', help='name of the Namespace AuthorizationRule')
+            c.argument('namespace_name', options_list=['--namespace-name'], id_part='name', help='name of the Namespace')
+
+    for scope in ['eventhubs namespace authorization-rule create', 'eventhubs namespace authorization-rule update', 'eventhubs eventhub authorization-rule create', 'eventhubs eventhub authorization-rule update']:
+        with self.argument_context(scope) as c:
+            c.argument('rights', arg_type=rights_arg_type)
+
+    with self.argument_context('eventhubs namespace authorization-rule keys renew') as c:
+        c.argument('key_type', arg_type=key_arg_type)
+
+
+# region - Eventhub Create
+    with self.argument_context('eventhubs eventhub') as c:
```

Comment:
`eventhubs eventhub` is redundant. The root should be `az eventhub` (singular) and the `eventhubs eventhub` commands should live under `eventhub`, just like with `webapp` and `functionapp`.

Reply:
@v-Ajnava Was there any decision on this?
Currently, it seems like this comment hasn't been addressed.

Reply:
Leave it as is. People can always use alias to make it whatever they want.

Reply:
had discussed with team and PM, they want it to be az eventhubs eventhub to avoid any  confusion in customer as eventhub is associated to namespace. also service is called eventhubs